source/js/foundation/pages/directory-listing-filters.js

```diff
@@ -48,6 +48,7 @@
 
       // Freeform fields may not contain HTML
       [`name`, `location`, `user_bio`].forEach((type) => {
+        if (!sanitizer) return;
```

Comment:
this feels impossible. If `document.createElement` doesn't work, the browser is broken...